deployment/src/main/java/io/quarkiverse/loggingmanager/deployment/LoggingManagerConfig.java

```diff
@@ -25,6 +25,13 @@
     @ConfigItem(defaultValue = "Loggers")
```

Comment:
What about changing the openapi tag to `Logging Manager` or similar?

Reply:
We had a problem in the passed with this having a space, that is why it's renames to Loggers. But we can maybe make the default Logging_Manager ? (see https://github.com/quarkiverse/quarkus-logging-manager/issues/17)

Reply:
I made it `Logging-manager`